packages/lib/server/defaultResponder.ts

```diff
@@ -15,8 +15,11 @@ function defaultResponder<T>(f: Handle<T>) {
       ok = true;
       if (result) res.json(result);
     } catch (err) {
-      console.error(err);
       const error = getServerErrorFromUnknown(err);
+      // user errors should not end up in stdout.
+      if (error.statusCode < 400 || error.statusCode >= 500) {
```

Comment:
There can be non-HTTP errors as well e.g. JSON parsing failure, that won't be logged this way.

Reply:
Can we move the login alotgether into `getServerErrorFromUnknown`? There at least we can know for example if it's a ZodError and we may want to log it to cover new edge-cases.

Reply:
@emrysal